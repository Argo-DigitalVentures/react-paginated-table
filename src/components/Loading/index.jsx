import React from 'react';

function Loading(props) {
  return (
    <div {...props}>Loading...</div>
  );
}

export default Loading;
