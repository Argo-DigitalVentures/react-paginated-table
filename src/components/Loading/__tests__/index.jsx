import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';

import Loading from '../';

describe('<Loading />', () => {
  it('should accept className, style, and other DOM props', () => {
    const tree = shallow(
      <Loading
        className="test"
        style={{
          color: 'green',
        }}
        aria-label="loading indicator"
      />,
    );
    expect(toJson(tree)).toMatchSnapshot();
  });
});
