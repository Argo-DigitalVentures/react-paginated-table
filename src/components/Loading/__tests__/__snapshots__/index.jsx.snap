exports[`<Loading /> should accept className, style, and other DOM props 1`] = `
<div
  aria-label="loading indicator"
  className="test"
  style={
    Object {
      "color": "green",
    }
  }>
  Loading...
</div>
`;
